kie-wb/pom.xml

```diff
@@ -33,6 +33,7 @@
   <modules>
     <module>kie-wb-webapp</module>
     <module>kie-wb-distribution-wars</module>
+    <module>kie-wb-webapp-test</module>
```

Comment:
Can you setup this in a new maven profile? Also make sure that it also runs on fullProfile